src/upload.ts

```diff
@@ -154,34 +153,32 @@ async function uploadVideo(videoJSON: Video, messageTransport: MessageTransport)
             if (progressChecker == undefined || !curProgress) return
             curProgress = curProgress.split(" ").find((txt: string) => txt.indexOf("%") != -1)
             let newProgress = curProgress ? parseInt(curProgress.slice(0, -1)) : 0
-            if ( progress.progress == newProgress ) return
+            if (progress.progress == newProgress) return
             progress.progress = newProgress
             videoJSON.onProgress!(progress)
         }, 500)
     }
 
     const errorMessage = await page.evaluate(() => (document.querySelector('.error-area.style-scope.ytcp-uploads-dialog') as HTMLElement)?.innerText.trim())
-
     if (errorMessage) {
         await browser.close()
         throw new Error('Youtube returned an error : ' + errorMessage)
     }
 
     // Wait for upload to complete
-    const uploadCompletePromise = page.waitForXPath('//*[contains(text(),"Upload complete")]', { timeout: 0 }).then(() => 'uploadComplete')
+    const uploadCompletePromise = page.waitForXPath('//tp-yt-paper-progress[contains(@class,"ytcp-video-upload-progress-hover") and @value="100"]', { timeout: 0 }).then(() => 'uploadComplete')
 
     // Check if daily upload limit is reached
-    const dailyUploadPromise = page.waitForXPath('//*[contains(text(),"Daily upload limit reached")]', { timeout: 0 }).then(() => 'dailyUploadReached');
-
+    const dailyUploadPromise = page.waitForXPath('//div[contains(text(),"Daily upload limit reached")]', { timeout: 0 }).then(() => 'dailyUploadReached');
     const uploadResult = await Promise.any([uploadCompletePromise, dailyUploadPromise])
     if (uploadResult === 'dailyUploadReached') {
-        await browser.close();
+        browser.close();
         throw new Error('Daily upload limit reached');
     }
 
     // Wait for upload to go away and processing to start, skip the wait if the user doesn't want it.
     if (!videoJSON.skipProcessingWait) {
-        await page.waitForXPath('//*[contains(text(),"Upload complete")]', { hidden: true, timeout: 0 })
+        await page.waitForXPath('//*[contains(text(),"Video upload complete")]', { hidden: true, timeout: 0 })
```

Comment:
Same question as https://github.com/fawazahmed0/youtube-uploader/pull/175/files#r1048469921

Reply:
I inspect the element in youtube video upload pannel , and find out no element contain "Upload complete",  only "Video upload complete" existed.
Currently not test with short video, as well